client/src/components/Login/Login.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import {
+  Form, Input, Button,
+} from 'antd';
+import axios from 'axios';
+import './Login.css';
+
+function Demo() {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const changeEmail = (e) => {
+    e.preventDefault();
+    setEmail(e.target.value);
+  };
+
+  const changePassword = (e) => {
+    e.preventDefault();
+    setPassword(e.target.value);
+  };
+
+  const onFinish = async () => {
+    await axios.post('/api/login', {
+      email,
+      password,
+    });
+  };
```

Comment:
You received here a message or error from the back so add try and catch and handle them so they appear to the user. And you still need to add the redux logic here